src/internal/deploy/assets/assets.go

```diff
@@ -730,6 +743,58 @@ func PachdDeployment(opts *AssetOpts, objectStoreBackend Backend, hostPath strin
 	}
 }
 
+// EnterpriseService returns a service for the Enterprise Server
+func EnterpriseService(opts *AssetOpts) *v1.Service {
```

Comment:
I think we have started pulling these functions out of `assets.go`; see `postgres.go` and `etcd.go`.  Can we make a separate file in the same package for this?